util/activity-report/developer-activity.js

```diff
@@ -87,26 +96,45 @@ async function queryDeveloperActivity(
   );
 
   return {
+    start_date: startDate,
+    end_date: endDate,
+    windows: windowsAmount,
     chain,
-    start_date: windows[0][0],
-    end_date: windows[0][1],
-    window_length_days: windowLength / 8.64e7, // ms per day
-    windows_amount: Number(windowsAmount),
     deployment_tx_count: {
       current: deployments[deployments.length - 1],
       sma: getMovingAverage('simple', deployments),
-      ema: getMovingAverage('exponential', deployments),
       wma: getMovingAverage('weighted', deployments),
     },
     deployment_account_count: {
       current: accounts[accounts.length - 1],
       sma: getMovingAverage('simple', accounts),
-      ema: getMovingAverage('exponential', accounts),
       wma: getMovingAverage('weighted', accounts),
     },
   };
 }
 
 module.exports = {
-  queryDeveloperActivity,
+  path: '/api/v1/rsk-activity-report/developer-activity',
+  cacheTtl: 600,
+  fetch,
+  queryStringParams: [
+    {
+      name: 'start-date',
+      validate: validateStartDate,
```

Comment:
💡 in future, if other APIs also have start date, end date, and date windows, can extract these into a common util file, similar to `validateChain`.

Reply:
sure, I was going to do this